inventory.yml.sample

```diff
@@ -179,6 +179,7 @@ all:
 
         http_store:
           ansible_host: 10.40.0.100
+          http_store_port: 80
```

Comment:
`http_store_port` should be in defaults. Users can add it here as an override but this shouldn't be the source of the var.

Reply:
As long as this change isn't going to break anything we can allow it. Afterwards take a pass and determine if any efficiencies can be added here.